airflow/api_fastapi/core_api/datamodels/dag_run.py

```diff
@@ -20,10 +20,11 @@
 from datetime import datetime
 from enum import Enum
 
+import pendulum
 from pydantic import AwareDatetime, Field, NonNegativeInt, model_validator
 
 from airflow.api_fastapi.core_api.base import BaseModel, StrictBaseModel
-from airflow.models import DagRun
+from airflow.models import DAG, DagRun
```

Comment:
nit: not sure whether we could move `DAG` to TYPE_CHECKING 🤔

Reply:
changed it